tests/functional/conftest.py

```diff
@@ -28,6 +27,11 @@
 from applitools.core import ServerConnector
 from tests.utils import get_session_results, send_result_report
 
+try:
+    import contextlib2 as contextlib
```

Comment:
I don't see this library in dependencies. Where you've added it?

Reply:
I don't see where you use this library and why do we need to add this extra dependency

Reply:
I somehow had it in dependencies, but surely need to make it explicit

Reply:
It's backport of modern contextlib to python2 where it's missing ExitStack that is used by spy

Reply:
If it required only in our tests, maybe we won't add extra dependencies into SDK? Sure it would install only with py2 but I don't like change that part of code and adding new dependencies 🤷🏻‍♂️

Reply:
I've added it to test dependencies, not SDK dependencies.
It's used by spy and turned out no tests using using were run in python2 on ci. I added it to be able to run them locally.
Or maybe we might make spy fixture only accessible on python3?